test/wdio_functional/spec/management/management.suite.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+
+
+
+describe('Management App', function () {
+  before(async function () {
+    try {
+      this.esArchiver = await global.getService('esArchiver');
+      this.logger = await global.getService('log');
+      await this.esArchiver.unload('logstash_functional');
+      await this.esArchiver.load('empty_kibana');
+      await this.esArchiver.loadIfNeeded('makelogs');
+    } catch (e) {
+      this.logger.error('Data load failed.');
+      this.logger.error(e);
+    }
+  });
+
+  describe('Index Pattern Tests', function () {
+    require('./index_pattern.spec');
```

Comment:
I'm getting the impression from this describe being here instead of in index_pattern.spec that you would add additional .spec files for more index pattern tests?  Is it 1 test per .spec file?  
So if there were multiple index pattern spec files required here in this describe, I wouldn't be able to easily tell which file contained the test case based on the fully-qualified describe and test name.
I know we have some amount of variation on how describes are done in the existing ftr so I'm just looking to make things as clear and consistent as possible on new work.

Reply:
@LeeDr In terms of adding these tests, I wanted to mirror what is in the FTR and also make sure we have the ability to pick which test files run in which order. Hence, this pattern. We could definitely add more tests to `./index_pattern.spec`. To me, in terms of what we have now, we organize tests by feature. We could change that to grouping tests by type: i.e. `smoke`, `sanity`, `regression`, etc. If we do that, then we could have a different subset run for PRs, run for releases, etc.

Reply:
But whichever one works. The reason I called that describe block indexpatterns instead of CreateIndexPatterns, is because I wanted to leave the door open to add more tests to this file that are involving index patterns.

Reply:
Ff you look at the existing visualize tests as an example, the index file has a describe `visualize app` which encompasses a before method and a bunch of tests;
https://github.com/elastic/kibana/blob/master/test/functional/apps/visualize/index.js#L26

And then each test file typically has a describe, like this one has `data table`;
https://github.com/elastic/kibana/blob/master/test/functional/apps/visualize/_data_table.js#L31

So in the test output (depending on the reporter style) if there were 2 tests in index_pattern.spec and 2 tests in some_other_index_pattern.spec, we would want a unique describe in each of those to group those tests.

Reply:
@LeeDr  Yep. Adding multiple describes within a spec app is fine. You can even add describes, within describes, within describes.